src/commands/synthetics/utils/internal.ts

```diff
@@ -255,12 +254,12 @@ export const validateAndParseOverrides = (overrides: string[] | undefined): Accu
           }
           break
 
-        // Convert to StringObject
+        // Convert to StringMap
         case 'headers':
         case 'variables':
           if (subKey) {
-            const stringMap: StringMap = (acc[key] = acc[key] ?? {}) // acc[key] could be undefined, because of potential js Proxy
-            stringMap[subKey] = value
+            acc[key] = acc[key] ?? {} // acc[key] could be undefined, because of potential js Proxy
```

Comment:
Now that this line is (almost) the same as the others, I don't understand why you are talking about "potential js Proxy" only on that line. Why could a `Proxy` be used, and how?

Reply:
Oups forgot to remove the comment 😁